pkg/controllers/provisioning/scheduling/preferences.go

```diff
@@ -100,6 +106,34 @@ func (p *Preferences) removePreferredNodeAffinityTerm(pod *v1.Pod) *string {
 	return nil
 }
 
+func (p *Preferences) removeRequiredNodeAffinityTerm(pod *v1.Pod) *string {
+	if pod.Spec.Affinity == nil ||
+		pod.Spec.Affinity.NodeAffinity == nil ||
+		pod.Spec.Affinity.NodeAffinity.RequiredDuringSchedulingIgnoredDuringExecution == nil ||
+		len(pod.Spec.Affinity.NodeAffinity.RequiredDuringSchedulingIgnoredDuringExecution.NodeSelectorTerms) == 0 {
+		return nil
+	}
+	terms := pod.Spec.Affinity.NodeAffinity.RequiredDuringSchedulingIgnoredDuringExecution.NodeSelectorTerms
+	// Remove the first term if there's more than one (terms are an OR semantic), Unlike preferred affinity, we cannot remove all terms
+	if len(terms) > 1 {
+		pod.Spec.Affinity.NodeAffinity.RequiredDuringSchedulingIgnoredDuringExecution.NodeSelectorTerms = terms[1:]
+		return ptr.String(fmt.Sprintf("removing: spec.affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution[0]=%s", pretty.Concise(terms[0])))
```

Comment:
It only gets logged if the pod has failed to schedule.  Most of the time this will never log anything. Preferences is a little odd , look at like 61 or so.  The first time it's called, it doesn't relax anything.